### PR TITLE
Use promu tool to build project

### PR DESCRIPTION
**Pull request by timonwong**

### CODE REVIEW

**Comment by timonwong** [comment]
Sorry

File: .travis.yml [hunk 1]
```diff
@@ -1,27 +1,15 @@
+sudo: false
+
 language: go
 go:
-  - 1.7
+  - 1.6.2
```
Comment (nordicdyno) [inline .travis.yml:5]: why just 1.6.2?

**Comment by timonwong** [comment]
@nordicdyno Ooops, please just ignore this pull request (it's only useful
for my own fork only), I opened this accidentally, sorry :(

**Comment by nordicdyno** [comment]
@timonwong do you have any plans to contribute your changes in this repo?

**Comment by timonwong** [comment]
@nordicdyno Of course, I've opened a new PR: #16

For the promtool stuff, I'm not sure if it's ok here, it need some extra configuration (environment variables) on circleci to make it work.

**Pull request closed without merging**